src/components/AppButton.js

```diff
@@ -0,0 +1,43 @@
+import { LitElement, html, css } from 'lit-element';
+
+class AppButton extends LitElement {
+
+  static get properties(){
+    return {
+      label: { type: String }
+    };
+  }
+  
+  static get styles() {
+    return css`
+      button {
+        cursor: pointer;
+        color: #fff;
+        border: 0;
+        padding: 8px 10px;
+        border-radius: 4px;
+      }
+
+      .green {
+        background-color: #28e236;
+      }
+
+      .red {
+        background-color: #e22828;
+      }
+    `;
+  }
+
+  constructor() {
+    super();
+  }
+
+  render() {
+    return html`
+      <button class="${this.color}" @click=${this.clickHandler}>${this.label}</button>
```

Comment:
We could have used the `classMap` or `styleMap` lit directive here. Start using them from the next time.

Also, better to add quotation marks in the props and handlers. For example: 

``` @click = "${this.clickHandler}" ```

Also, use the convention to name the function names as Verbs and props as Nouns.

For example:

Method: `handleClick` instead of `clickHandler`